pkg/server/app/app_test.go

```diff
@@ -460,6 +470,43 @@ func TestAppOperationsInfoErrNotFound(t *testing.T) {
 	}
 }
 
+func TestAppOperationsList(t *testing.T) {
+	tops := team.NewFakeOperations()
+	ops := NewOperations(tops, &fakeK8sOperations{}, nil)
+	teamName := "luizalabs"
+	user := &storage.User{Email: "teresa@luizalabs.com"}
+	app := &App{Name: "teresa", Team: teamName}
+	tops.(*team.FakeOperations).Storage[app.Name] = &storage.Team{
+		Name:  teamName,
+		Users: []storage.User{*user},
+	}
+
+	list, err := ops.List(user)
+	if err != nil {
+		t.Fatal("error getting app list: ", err)
+	}
+
+	for _, a := range list {
+		if a.Team != teamName {
+			t.Errorf("expected %s, got %s", teamName, a.Team)
+		}
+
+		if len(a.Addresses) != 1 { // see fakeK8sOperations.AddressList
+			t.Errorf("expected 2, got %d", len(a.Addresses))
+		}
+	}
+}
+
+func TestAppOperationsListErrNotFound(t *testing.T) {
+	tops := team.NewFakeOperations()
+	ops := NewOperations(tops, &errK8sOperations{Err: ErrNotFound}, nil)
```

Comment:
Should be an empty list and this meaning that "not found" any app. Or am I wrong?